src/dal/views.py

```diff
@@ -49,3 +59,27 @@ def get_queryset(self):
             qs = qs.filter(name__icontains=self.q)
 
         return qs
+
+    def post(self, request):
+        """Create an object given a text, for staff only."""
+        if not request.user.is_authenticated() and not request.user.is_staff:
+            # We should have security by default, but this should be somehow
+            # configurable. Perhaps check django.contrib.auth.permissions or
+            # reuse something that comes from django.contrib.auth ?
+            return http.HttpResponseForbidden()
+
+        if not self.create_field:
+            return http.HttpResponseForbidden()
```

Comment:
If `create_field` is not set, then the autocomplete should not even post to this view. That said, ImproperlyConfigured is better, thanks !